internal/api/scim/resources/user.go

```diff
@@ -240,7 +240,7 @@ func (h *UsersHandler) List(ctx context.Context, request *ListRequest) (*ListRes
 		return NewListResponse(count, q.SearchRequest, make([]*ScimUser, 0)), nil
 	}
 
-	users, err := h.query.SearchUsers(ctx, q, nil)
+	users, err := h.query.SearchUsers(ctx, q, "", nil)
 	if err != nil {
```

Comment:
SCIM requests actually happen in the scope of the organization, but it is parsed elsewhere:

https://github.com/zitadel/zitadel/blob/18ce29b62827b6ccf866c2a5585df22a1ae2bc97/internal/api/scim/resources/user_query_builder.go#L78-L83

In this case I suggest passing the context data directly:


```suggestion
	users, err := h.query.SearchUsers(ctx, q, authz.GetCtxData(ctx).OrgID, nil)
```